services/core-web/common/selectors/noticeOfWorkSelectors.js

```diff
@@ -22,20 +22,22 @@ export const getNOWReclamationSummary = createSelector(
     const reclamationList = [];
     if (options.length > 0) {
       options.forEach(({ value, label }) => {
+        // options uses the value of the code table to map to the response_model names for activities, they all match except camps.
+        const newValue = value === "camp" ? "camps" : value;
```

Comment:
can we rename value to camps on DB level instead? 

Reply:
I was considering that - I don't want to rename the code table as "camp" is the code, but I could rename the response_model key. 

This was just less effort and no negative side effects - but I can rename the response_model key and test everything if that is preferred. 

Let me know

Reply:
Discussed: Opted to rename the response_model key.